code/game/machinery/computer/medical.dm

```diff
@@ -490,28 +547,6 @@
 				if((istype(src.active2, /datum/data/record) && src.active2.fields[text("com_[]", href_list["del_c"])]))
 					src.active2.fields[text("com_[]", href_list["del_c"])] = "<B>Deleted</B>"
 
-			else if(href_list["search"])
-				var/t1 = stripped_input(usr, "Search String: (Name, DNA, or ID)", "Med. records")
-				if(!canUseMedicalRecordsConsole(usr, t1))
-					return
-				src.active1 = null
-				src.active2 = null
-				t1 = lowertext(t1)
-				for(var/datum/data/record/R in GLOB.data_core.medical)
-					if((lowertext(R.fields["name"]) == t1 || t1 == lowertext(R.fields["id"]) || t1 == lowertext(R.fields["b_dna"])))
-						src.active2 = R
-					else
-						//Foreach continue //goto(3229)
```

Comment:
bruh

Reply:
Huh? The search option was super shit, and since this PR implements a filter I don't see any reason to keep it in the code.